src/main/java/uk/gov/ons/fwmt/job_service/service/impl/CSVParsingServiceImpl.java

```diff
@@ -124,45 +122,29 @@ public LocalDateTime convertToGFFDate(String stage) {
       assert month >= 1 && month < 12;
       LocalDate initial = LocalDate.of(year, month, 1);
       LocalDate endOfMonth = initial.withDayOfMonth(initial.lengthOfMonth());
-      return endOfMonth.atTime(23, 59, 59);
+      return endOfMonth;
     }
   }
 
   // technically, 'stage' here is the field period 'fp'
   // TODO double check to ensure that this is correct
-  public Date convertToLFSDate(String stage) {
-    if (fieldPeriodResourceService.existsByFieldPeriod(stage)) {
-      Calendar cal = Calendar.getInstance();
-      final FieldPeriodDto fieldPeriod = fieldPeriodResourceService.findByFieldPeriod(stage).get();
-      cal.setTime(fieldPeriod.getEndDate());
-      cal.set(Calendar.HOUR, 11);
-      cal.set(Calendar.MINUTE, 59);
-      cal.set(Calendar.SECOND, 59);
-      cal.set(Calendar.AM_PM, Calendar.PM);
-      return cal.getTime();
+  public LocalDate convertToLFSDate(String stage) {
+    final Optional<FieldPeriodDto> existsByFieldperiod = fieldPeriodResourceService.findByFieldPeriod(stage);
+    if (existsByFieldperiod.isPresent()) {
+      final FieldPeriodDto fieldPeriod = existsByFieldperiod.get();
+      return fieldPeriod.getEndDate();
     } else {
       // BACKUP!!! THIS SHOULD NOT HAPPEN
       int year = 2010 + Integer.parseInt(stage.substring(0, 1));
       int quarter = Integer.parseInt(stage.substring(1, 2));
       int week = stage.toLowerCase().charAt(2) - 'a' + 3;
-      Calendar cal = Calendar.getInstance();
-      cal.set(2010 + year, 1 + (3 * (quarter - 1)) - 1, 1);
-      cal.set(Calendar.HOUR, 11);
-      cal.set(Calendar.MINUTE, 59);
-      cal.set(Calendar.SECOND, 59);
-      cal.set(Calendar.AM_PM, Calendar.PM);
-      cal.add(Calendar.DATE, (7 * week) - 1);
-      return cal.getTime();
+      int month = 1 + (3 * (quarter - 1));
+      int day = (7 * week) - 1;
```

Comment:
This is currently wrong, but is not currently required.

Reply:
updated the todo above the method to mention this